src/main/java/com/julienvey/trello/TrelloHttpClient.java

```diff
@@ -5,9 +5,11 @@
 public interface TrelloHttpClient {
     <T> T get(String url, Class<T> objectClass, String... params);
 
-    <T> T postForObject(String url, T object, Class<T> objectClass, String... params);
+    <T> T postForObject(String url, Object object, Class<T> objectClass, String... params);
```

Comment:
was there a very solid reason to remove this type check?

Reply:
Yes, sure. It's a very serious limitation for `TrelloHttpClient` to require same response type as request body type was. For example the user cannot preform following invocation:

```
Map<String, String> body = new HasMap<>();
Class<Member> responseType = Member.class;

httpClient.postForObject(url, body, responseType); // compilation error
```

Basically, we can't send request with differing request and response types. The Spring's `RestTemplate` does use `Object` not the `T` type for request body for these reason.

Reply:
ok, sounds good